platform/ios/app/MBXViewController.m

```diff
@@ -1274,12 +1274,7 @@ - (void)styleRasterSource
 
 -(void)styleCountryLabelsLanguage
 {
-    NSArray<NSString *> *labelLayers = @[
-        @"country-label-lg",
-        @"country-label-md",
-        @"country-label-sm",
-    ];
-    [self styleLabelLanguageForLayersNamed:labelLayers];
+    [self.mapView.style setLocalizeLabels:YES];
```

Comment:
Along the lines of [this comment](https://github.com/mapbox/mapbox-gl-native/pull/9582/files#r129494333), this should be converted to an on/off toggle.